integration-tests/test_general.py

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+
+import unittest
+from smart_open import smart_open_lib
+
+
+class Test(unittest.TestCase):
+
+
+    def test_host_name(self):
```

Comment:
Not a suitable place for the test, this isn't an integration test, please add your test to existing files in `smart_open/tests/`